.travis.yml

```diff
@@ -28,17 +28,36 @@ before_install:
         conda config --set always_yes yes --set changeps1 no
         conda update -q conda
         conda info -a
-        conda create -q -n p4env python=$TRAVIS_PYTHON_VERSION ci-psi4 psi4 numpy matplotlib jupyter scipy -c psi4/label/dev -c defaults -c conda-forge
+        conda create -q -n p4env python=$TRAVIS_PYTHON_VERSION ci-psi4 psi4 numpy matplotlib jupyter scipy cython -c psi4/label/dev -c defaults -c conda-forge -c anaconda
 
         source activate p4env
         conda install pytest pytest-cov codecov -c conda-forge
         pip install pytest-shutil
         conda list
+        pip install git+https://github.com/theochem/iodata.git
     else
         python -m pip install -U pytest pytest-cov
     fi
+    if [[ $(echo "$TRAVIS_PYTHON_VERSION < 3.0" | bc -l) == 1 ]];
```

Comment:
This is all the same as in #873 - could you separate it out into a smaller separate PR?

Reply:
@langner I am a bit confused here -- I thought that the changes in Travis should be included in the PR so that there can be tests run for the new methods? If you meant that the Travis configurations / test functions should be a separate PR and the bridge functions in another PR, I can definitely work it out tomorrow. Thanks!

Reply:
Yeah, if it's something that's useful in both PRs and can exist in master before either of them, why not merge it in earlier? Trimming down your PRs is always good.